app/src/main/java/com/andrew/tasky/presentation/task_detail/TaskDetailViewModel.kt

```diff
@@ -1,6 +1,37 @@
 package com.andrew.tasky.presentation.task_detail
 
 import androidx.lifecycle.ViewModel
+import com.andrew.tasky.util.TaskType
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import java.text.SimpleDateFormat
+import java.util.*
 
 class TaskDetailViewModel : ViewModel() {
+
+    private val calendar = Calendar.getInstance()
+    private val _selectedDate = MutableStateFlow(SimpleDateFormat("MMM dd yyyy", Locale.ENGLISH).format((calendar.time)))
```

Comment:
I'd recommend to take a look at the Java Date Time API instead of `SimpleDateFormat` which offers much more cool functionality (like calculating with dates, adding/subtracting days, months etc.) and it's thread-safe (which `SimpleDateFormat` is not: https://www.callicoder.com/java-simpledateformat-thread-safety-issues/#:~:text=One%20of%20the%20most%20important,environments%20if%20not%20used%20properly.)